### DISCUSSION TITLE
Display Error maybe. My account shows wrong number of repositories

### DISCUSSION THREAD

**Discussion by ascyrax** [community/community#23127] [New to GitHub]
<p><img src="https://user-images.githubusercontent.com/51852712/181074994-7e8d3c5b-fc1b-4434-a743-1c8fe7a762ad.png" alt="image" data-base62-sha1="2HPV2mtZ30OWGyZ4E5INu7aRTDv" width="681" height="158"></p>
<p>I have just 9 repositories but as you can see in the image above, it shows 93 to me(JUST ME). Its been like this for about a week now.<br>
Is this some kind of bug or maybe I am missing something?<br>
<a href="https://github.com/ascyrax?tab=repositories" rel="noopener nofollow ugc">my github link</a><br>
PS:- I used to have 93 repositories till about a week ago, but then I deleted the useless ones.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>I’ve seen a few reports like this over the last few days, so I suspect there’s some cache that’s slow to update. In the thread linked below someone mentioned the problem went away after they created a new repository. :sweat_smile:</p>
<aside class="quote quote-modified" data-post="1" data-topic="256694">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/more-repositories-shown-than-i-have-solved/256694">More Repositories shown than I have [solved]</a> <a class="badge-wrapper  bullet" href="/c/github-help/48"><span class="badge-category-bg" style="background-color: #6a737d;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Ask questions about your GitHub account and the core features of GitHub.">GitHub Help</span></a>
  </div>
  <blockquote>
    Hey all wave 
My account shows 16 Repositories in the upper menu bar, but as a matter of fact I do only have 9. 
Can anybody shed some light for me? blush 
 <a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/3/0378980e2879e87167a2b63cd0890309e9f5fc6e.png" data-download-href="/uploads/short-url/uHNPM8pdyboKNYDKjRgsBqMkaO.png?dl=1" title="image" rel="noopener nofollow ugc">[image]</a>
  </blockquote>
</aside>

<p>So I guess either do that, or send a bug report to <a href="https://support.github.com/request" class="inline-onebox">GitHub Support</a> so they can get on a proper fix. :slightly_smiling_face:</p>

**Comment by ascyrax** [comment]
<p>Thanks a lot. That worked.</p>
